src/account-creation.ts

```diff
@@ -0,0 +1,100 @@
+import {Checksum256, Checksum256Type, NameType, PermissionLevel, PublicKey} from '@wharfkit/antelope'
+import { LocaleDefinitions } from '.'
+
+/**
+ * The static configuration of an [[AccountCreationPlugin]].
+ */
+export interface AccountCreationPluginConfig {
+    /**
+     * Indicates if the pp requires the user to manually select the blockchain to authorize against.
+     */
+    requiresChainSelect: boolean
+    /**
+     * Indicates if the [[AccountCreationPlugin]] requires the user to select an account name to use from a list.
+     */
+    requiresAccountNameSelect?: boolean
```

Comment:
Should this be requires account name input, or removed?

Reply:
Yeah, I just ended up removing this.